src/sentry/utils/performance_issues/detectors/consecutive_db_detector.py

```diff
@@ -261,7 +261,7 @@ def is_creation_allowed_for_project(self, project: Project) -> bool:
         return self.settings["detection_enabled"]
 
     @classmethod
-    def is_event_eligible(cls, event, project: Project | None = None) -> bool:
+    def is_event_eligible(cls, event: dict[str, Any], project: Project | None = None) -> bool:
```

Comment:
What's the current meta for Python typing? `dict[str, Any]` or `Mapping[str, Any]`?

Reply:
https://typing.readthedocs.io/en/latest/source/best_practices.html#arguments-and-return-types

though generally these are easy enough to be concrete types since we are not dealing with variance and we are not providing a public api

in theory we should be serializing event dicts somewhere so they're not just a big spooky bag of unknown attributes -- but that's probably a bigger project